frontend/src/features/admin-form/create/builder-and-design/BuilderAndDesignContent/BuilderAndDesignContent.tsx

```diff
@@ -15,69 +29,158 @@ import BuilderAndDesignPlaceholder from './BuilderAndDesignPlaceholder'
+            {form?._id ?? 'Form Identification Number'}
+            <br />
+            {format(new Date(), 'dd MMM yyyy, h:m aa')}
+          </Text>
+
+          <Flex pt="1.75rem" gap="2rem">
+            <Button>Save this response</Button>
+            <Button variant="clear">{endPageData.buttonText}</Button>
+          </Flex>
+        </Box>
+      </Stack>
+    </Flex>
+  )
+}
 interface BuilderAndDesignContentProps {
   placeholderProps: DndPlaceholderProps
 }
 
 export const BuilderAndDesignContent = ({
   placeholderProps,
 }: BuilderAndDesignContentProps): JSX.Element => {
-  const setFieldsToInactive = useBuilderAndDesignStore(setToInactiveSelector)
-  const { builderFields } = useBuilderFields()
-  const { handleBuilderClick } = useCreatePageSidebar()
+  const { stateData, setToInactive: setFieldsToInactive } =
+    useBuilderAndDesignStore(
+      useCallback(
+        (state) => ({
+          stateData: stateDataSelector(state),
+          setToInactive: setToInactiveSelector(state),
+        }),
+        [],
+      ),
+    )
 
   useEffect(() => setFieldsToInactive, [setFieldsToInactive])
 
   return (
     <>
       <Flex flex={1} bg="neutral.200" overflow="auto">
-        <Flex
-          m={{ base: 0, md: '2rem' }}
-          mb={0}
-          flex={1}
-          bg={{ base: 'secondary.100', md: 'primary.100' }}
-          p={{ base: '1.5rem', md: '2.5rem' }}
-          justify="center"
-          overflow="auto"
-        >
-          <Flex
-            h="fit-content"
-            bg="white"
-            p={{ base: 0, md: '2.5rem' }}
-            maxW="57rem"
-            w="100%"
-            flexDir="column"
-          >
-            <Droppable droppableId={FIELD_LIST_DROP_ID}>
-              {(provided, snapshot) =>
-                builderFields?.length ? (
-                  <Box
-                    pos="relative"
-                    ref={provided.innerRef}
-                    {...provided.droppableProps}
-                  >
-                    <BuilderFields
-                      fields={builderFields}
-                      isDraggingOver={snapshot.isDraggingOver}
-                    />
-                    {provided.placeholder}
-                    <BuilderAndDesignPlaceholder
-                      placeholderProps={placeholderProps}
-                      isDraggingOver={snapshot.isDraggingOver}
-                    />
-                  </Box>
-                ) : (
-                  <EmptyFormPlaceholder
-                    ref={provided.innerRef}
-                    {...provided.droppableProps}
-                    isDraggingOver={snapshot.isDraggingOver}
-                    onClick={handleBuilderClick}
-                  />
-                )
-              }
-            </Droppable>
-          </Flex>
-        </Flex>
+        {stateData.state === BuildFieldState.EditingEndPage ? (
+          <EndPageView />
+        ) : (
+          <FormBuilder placeholderProps={placeholderProps} />
+        )}
```

Comment:
instead of rendering between end page and builder view, should we just render both and hide them appropriately? This allows for the state and scroll position  of either previews to be maintained

Reply:
Good idea, I've updated the PR to do that, thanks